tests/io/dask/delayed/test_delete.py

```diff
@@ -17,3 +20,8 @@ def _delete(*args, **kwargs):
 @pytest.fixture
 def bound_delete_dataset():
     return _delete
+
+
+@pytest.fixture()
```

Comment:
Why is this is fixture here used?

Reply:
for the dask tasks, I have used dask garbage collection function.. since delete pipelines in the code which calls gc

Reply:
@mganesh1308 The point is that this fixture is not used anywhere. Please remove it again